src/serac/infrastructure/cli.cpp

```diff
@@ -29,6 +29,8 @@ std::unordered_map<std::string, std::string> defineAndParse(int argc, char* argv
                "Writes Sphinx documentation for input file, then exits");
   bool output_fields{true};
   app.add_flag("--output-fields,!--no-output-fields", output_fields, "Writes field data to file system.");
+  std::string output_directory;
+  app.add_option("-o, --output_directory", output_directory, "Directory to put outputted files.");
```

Comment:
Would the current working directory be a good default here?  It would have to be at the user's own risk if they're running multiple instances of serac simultaneously (which is what we ran into with the tests but seems like maybe a rare use case outside of that)

Reply:
Changed to input file "stem" (this means "basename - file extension" per std::filesystem::path::stem) per webex converstation.